### ISSUE TITLE
service coverage: DynamoDB

### ISSUE THREAD

**Issue by justinmk3** [aws/aws-toolkit-vscode#1421]
Support DynamoDB:

- List/Browse
- Delete
- Edit

Related: #476

**Comment by WooSung-Jung** [comment]
I want this Support.

**Comment by justinmk3** [comment]
@WooSung-Jung Thanks for your feedback, please add your upvote 👍  to the main description so it will be counted when comparing requests in this project.

**Comment by luca-terrazzan-docebo** [comment]
It would be incredibly useful to also include batch inserts by selecting a file, a common functionality that not even the console itself supports.

**Comment by bryceitoc9** [comment]
Hi @luca-terrazzan-docebo , just making sure, are you talking about implementing [`BatchWriteItem`](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_BatchWriteItem.html) support or do you have something else in mind?

**Comment by luca-terrazzan-docebo** [comment]
@bryceitoc9 yes the underlying implementation can leverage the `BatchWriteItem`. 
For a bit of context: sometimes, during development, it is useful to download a set of records, massively alter them with a text editor and reupload them. Unfortunately, the AWS console does not support this scenario. It would be cool to have it done in VSCode directly!

**Comment by sliedig** [comment]
+1 to this.

**Comment by bogdartysh** [comment]
As for me even partial would be great

**Comment by jalbertxd** [comment]
+1

**Comment by o-alexandrov** [comment]
It'd be great if `AWS Toolkit` extension supported all of the features from another already existing VSCode extension:
- [Serverless Console](https://marketplace.visualstudio.com/items?itemName=devAdvice.serverlessconsole)

**Comment by speedhawk21** [comment]
Please add this to your next sprint plan.

**Comment by sushanth-octane** [comment]
Any update on this?
